tardis/model/parse_input.py

```diff
@@ -723,10 +687,8 @@ def calculate_t_radiative_from_t_inner(geometry, packet_source):
     Quantity
         The calculated radiative temperature.
     """
-    lambda_wien_inner = const.b_wien / packet_source.temperature
-    t_radiative = const.b_wien / (
-        lambda_wien_inner
-        * (1 + (geometry.v_middle - geometry.v_inner_boundary) / const.c)
+    t_radiative = packet_source.temperature / (
+        (1 - (geometry.v_middle - geometry.v_inner_boundary) / const.c)
```

Comment:
The documentation is incorrect here: the `1 - ` should be `1 + ` because temperature should *decrease* further out. The other change (packet_source.temperature) is fine.

Reply:
You can modify the documentation to fix the sign of the function

Reply:
Yes !!! That makes sense. I will change the documentation and will redo this change to the original.